frontend/src/components/input.tsx

```diff
@@ -60,9 +72,14 @@ export const Input = ({ sendMessage, waiting, suggestions }: InputProps) => {
     setUploadInProgress(true);
 
     try {
-      const { filename, id } = await uploadFileToServer(file);
-      console.log(`File uploaded successfully: ${filename} with id ${id}`);
+      const { filename, report } = await uploadFileToServer(file);
       setUploadedFile(file);
+      appendMessage(
+        { answer: 'Your report is ready to view.' },
```

Comment:
Can we add the file name in, something like: "Your ESG report for `file_name` is ready to view."